tools/strict-null-checks/write-tsconfig.js

```diff
@@ -1,12 +1,20 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 const fs = require('fs');
+const path = require('path');
+const prettier = require('prettier');
 
 module.exports = {
-    writeTsconfigSync: (tsconfigPath, content) => {
+    writeTsConfig: async (tsconfigPath, content) => {
         let serializedContent = JSON.stringify(content, null, '    ');
-        serializedContent += '\n';
 
-        fs.writeFileSync(tsconfigPath, serializedContent);
+        let prettierConfigPath = path.join(__dirname, '..', '..', 'prettier.config.js');
```

Comment:
Without this chance, `yarn null:autoadd` would try to write out the `include:` property on multiple lines and conflict with prettier wanting it on one line